src/stub.py

```diff
@@ -182,7 +183,10 @@ def install_files(self, cur_os):
         except:
             print("Failed to apply extended attributes, logo will not work.")
 
-        shutil.copy("m1n1.macho", os.path.join(self.osi.system))
+        if split_ver(cur_os.version) < (12, 1):
```

Comment:
This shouldn't be cur_os, it should be checking the version of the stub OS to be installed, since that is the kmutil that will end up being used on 12.0+

Reply:
yes, I was confused by the non-paired recovery for macOS 11.x. fixed